plugins/MauticFocusBundle/Form/Type/FocusType.php

```diff
@@ -123,6 +127,10 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             ]
         );
 
+        $websiteValue = $options['data']->getWebsite();
+        if (empty($websiteValue)) {
+            $websiteValue = $this->coreParametersHelper->get('site_url') . $this->router->generate('mautic_focus_index', ['page' => 1]);
```

Comment:
```suggestion
            $websiteValue = $this->router->generate('mautic_focus_index',
            ['page' => 1],
            \Symfony\Component\Routing\Generator\UrlGeneratorInterface::ABSOLUTE_URL
        );
```
Instead of manual concatenation, we can use Symfony's router to generate a fully-qualified URL.